yt_dlp/extractor/brightcove.py

```diff
@@ -577,6 +577,22 @@ def build_format_id(kind):
         if duration is not None and duration <= 0:
             is_live = True
 
+        common_res = ["160x90", "320x180", "480x270", "640x360", "768x432", "1024x576", "1280x720", "1366x768", "1920x1080"]
```

Comment:
makes more sense to do
```suggestion
        common_res = [(160, 90), (320, 180), ...]
```